modules/ztp-setting-managed-bare-metal-host-kernel-parameters.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-sites.adoc
+
+[id="setting-managed-bare-metal-host-kernel-parameters_{context}"]
+= Setting managed bare-metal host kernel parameters
+
+Specify kernel parameters for managed bare-metal hosts to customize the host's boot process. This is useful for clusters with specific environmental requirements. For example, specify the `rd.net.timeout.carrier` kernel parameter to facilitate static networking or to receive a DHCP address before downloading the root file system. 
+
+[NOTE]
+====
+In {product-title} {product-version}, you can only add kernel parameters. You can not replace or delete kernel parameters.
```

Comment:
Is this correct? It's based on this comment but unsure if I am understanding this correctly: https://github.com/openshift/assisted-service/pull/4669/files#diff-2ba87ee0005a7fb9cb20a9a3557045068d8e51e358692ed2cb52cb2feca321e0R204

Reply:
Yes